flutter/example/integration_test/integration_test.dart

```diff
@@ -162,66 +163,75 @@ void main() {
     await transaction.finish();
   });
 
-  // group('e2e', () {
-  //   var output = find.byKey(const Key('output'));
-  //   late Fixture fixture;
-  //
-  //   setUp(() {
-  //     fixture = Fixture();
-  //   });
-  //
-  //   testWidgets('captureException', (tester) async {
-  //     await setupSentryAndApp(tester,
-  //         dsn: exampleDsn, beforeSendCallback: fixture.beforeSend);
-  //
-  //     await tester.tap(find.text('captureException'));
-  //     await tester.pumpAndSettle();
-  //
-  //     final text = output.evaluate().single.widget as Text;
-  //     final id = text.data!;
-  //
-  //     final uri = Uri.parse(
-  //       'https://sentry.io/api/0/projects/$org/$slug/events/$id/',
-  //     );
-  //     expect(authToken, isNotEmpty);
-  //
-  //     final event = await fixture.poll(uri, authToken);
-  //     expect(event, isNotNull);
-  //
-  //     final sentEvent = fixture.sentEvent;
-  //     expect(sentEvent, isNotNull);
-  //
-  //     final tags = event!["tags"] as List<dynamic>;
-  //
-  //     expect(sentEvent!.eventId.toString(), event["id"]);
-  //     expect("_Exception: Exception: captureException", event["title"]);
-  //     expect(sentEvent.release, event["release"]["version"]);
-  //     expect(
-  //         2,
-  //         (tags.firstWhere((e) => e["value"] == sentEvent.environment) as Map)
-  //             .length);
-  //     expect(sentEvent.fingerprint, event["fingerprint"] ?? []);
-  //     expect(
-  //         2,
-  //         (tags.firstWhere((e) => e["value"] == SentryLevel.error.name) as Map)
-  //             .length);
-  //     expect(sentEvent.logger, event["logger"]);
-  //
-  //     final dist = tags.firstWhere((element) => element['key'] == 'dist');
-  //     expect('1', dist['value']);
-  //
-  //     final environment =
-  //         tags.firstWhere((element) => element['key'] == 'environment');
-  //     expect('integration', environment['value']);
-  //   });
-  // });
+  group('e2e', () {
+    var output = find.byKey(const Key('output'));
+    late Fixture fixture;
+
+    setUp(() {
+      fixture = Fixture();
+    });
+
+    testWidgets('captureException', (tester) async {
+      late Uri uri;
+      await restoreFlutterOnErrorAfter(() async {
```

Comment:
Looks like something for setup/teardown functions?

Reply:
I think it needs to be restored before expect calls are made, wouldn't work in setup or teardown

I'll double check

Reply:
ah, if that's the case, could you please add a comment to the function so anyone coming back to this won't have the same idea as I've just had?